splinepy/utils/data.py

```diff
@@ -218,14 +218,12 @@ def enforce_contiguous(array, dtype=None):
     -------
     contiguous_array: array-like
     """
-    if isinstance(array, np.ndarray):
+    if issubclass(type(array), np.ndarray):
```

Comment:
I'd keep is instance

Reply:
OK, I just remembered a conversation we had, I wasn't sure about the outcome anymore. Changed it back

Reply:
thank you!